tests/publiccloud/storage_perf.pm

```diff
@@ -202,7 +203,9 @@ sub run {
             my $db = get_var('PUBLIC_CLOUD_PERF_DB', 'perf_2');
             my $token = get_required_var('_SECRET_PUBLIC_CLOUD_PERF_DB_TOKEN');
             my $org = get_var('PUBLIC_CLOUD_PERF_DB_ORG', 'qec');
-            influxdb_push_data($url, $db, $org, $token, $data) if (check_var('PUBLIC_CLOUD_PERF_PUSH_DATA', 1));
+            $res = influxdb_push_data($url, $db, $org, $token, $data, proceed_on_failure => 1, timeout => $db_timeout)
+              if (check_var('PUBLIC_CLOUD_PERF_PUSH_DATA', 1));
```

Comment:
```suggestion
              if (check_var('PUBLIC_CLOUD_PERF_PUSH_DATA', 0));
```
I would not push data by default to avoid unintended data points in the DB.

Reply:
I didn't changed that `PUSH` parameter value: I only moved that `if` 1 line down, being too long and too far in one line, after I added more parameters to the routine call.
Moreover, changing now that value, we change the test logic even much more.

Reply:
please keep it in one line. This does not look pretty :smile:

Reply:
Sorry, you are right, I misread `check_var` for `get_var`. Your solution is correct.